src/d_iwad.c

```diff
@@ -602,11 +577,8 @@ void BuildIWADDirList(void)
     }
 
     // [FG] Add plain HOME directory
-    env = M_getenv("HOME");
-    if (env != NULL)
-    {
-        AddIWADDir(env);
-    }
+    env = M_HomeDir();
```

Comment:
This will behave differently on Windows. What's the closest pendant to `HOME` on Windows, `%USERPROFILE%`?

Reply:
I prefer not to use the profile directory in Windows. “Serious” programs and commercial games use it, but I like simple conservative Doom ports to only create files in the port directory.

I think only GZDoom uses the profile directory in Windows, and one of the popular user questions is how to disable that. They added it because commercial GZDoom based games require it.

Reply:
Just realized that the sole slash is entirely meaningless on Windows as a path string, and that it's thus okay to return it unconditionally as it's about the same as not returning any path string at all.